pytest.ini

```diff
@@ -4,3 +4,5 @@ addopts = --ff --nomigrations
 env =
     OPBEAT_DISABLE_SEND=true
```

Comment:
We removed Opbeat githooks. Not sure we still need this setting since the package is deprecated anyway. We could do away with it now I think or we can have that in another PR.